modules/persistent-storage-csi-efs-sts.adoc

```diff
@@ -88,6 +88,13 @@ $ ccoctl aws create-iam-roles --name my-aws-efs --credentials-requests-dir credr
 +
 You will need the role ARN when you install the AWS EFS CSI Driver Operator.
 
+. Create a secret by running the following command:
```

Comment:
Hi @lpettyjo 
I guess this step is not required, since after installing efs operator using ROLE ARN value it automatically creates secrets aws-efs-cloud-credentials in ns openshift-cluster-csi-drivers. Tested on 4.20 cluster OCP STS cluster. 
cc: @Phaow

Reply:
IIRC, the olm support install operators with `ROLE ARN`(standard way) on sts is from 4.14(https://issues.redhat.com//browse/STOR-1347)? The extra step seems only needed on the versions(4.13 and below) do not support install with `ROLE ARN`, in addition it seems we already have it -> https://docs.redhat.com/en/documentation/openshift_container_platform/4.13/html/storage/using-container-storage-interface-csi#efs-sts_persistent-storage-csi-aws-efs.